### ISSUE TITLE
There has a bug for exporting the model to onnx and a question for funasr_onnx lib

### ISSUE THREAD

**Issue by cx16528** [FunAudioLLM/SenseVoice#35]
First there has a bug in export_meta.py 
speech = speech.to(device=kwargs["device"])
speech_lengths = speech_lengths.to(device=kwargs["device"])
Using the "model.export()" fucntion, the error said kwargs cannot find the key "device", annotating this code will working. The tensor textnorm_query and language_query need to unsqueeze the second dimension for concatenating with other tensor.
```python
def export_forward(
    self,
    speech: torch.Tensor,
    speech_lengths: torch.Tensor,
    language: torch.Tensor,
    textnorm: torch.Tensor,
    **kwargs,
):

    # speech = speech.to(device=kwargs["device"])
    # speech_lengths = speech_lengths.to(device=kwargs["device"])

    language_query = self.embed(language).to(speech.device)

    textnorm_query = self.embed(textnorm).to(speech.device)
    textnorm_query = torch.unsqueeze(textnorm_query, dim=1)
    speech = torch.cat((textnorm_query, speech), dim=1)
    speech_lengths += 1

    event_emo_query = self.embed(torch.LongTensor([[1, 2]]).to(speech.device)).repeat(
        speech.size(0), 1, 1
    )
    language_query = torch.unsqueeze(language_query, dim=1)
    input_query = torch.cat((language_query, event_emo_query), dim=1)
    speech = torch.cat((input_query, speech), dim=1)
    speech_lengths += 3

    # Encoder
    encoder_out, encoder_out_lens = self.encoder(speech, speech_lengths)
    if isinstance(encoder_out, tuple):
        encoder_out = encoder_out[0]

    # c. Passed the encoder result and the beam search
    ctc_logits = self.ctc.log_softmax(encoder_out)


    return ctc_logits, encoder_out_lens

Second, I cannot find the code of sensevoice function in funasr_onnx lib, and my funasr_onnx edition is 0.3.1 that is newest that I can find the latest version. When you will update the next version for support the inference the onnx model.

**Comment by LauraGPT** [comment]
We are working on it now.

**Comment by LauraGPT** [comment]
Now ONNX and Libtorch is added: https://github.com/FunAudioLLM/SenseVoice?tab=readme-ov-file#onnx
